src/Amazon.Lambda.Tools/Commands/DeployServerlessCommand.cs

```diff
@@ -337,8 +335,7 @@ protected override async Task<bool> PerformActionAsync()
             }
 
             // The change set can take a few seconds to be reviewed and be ready to be executed.
-            if (!await WaitForChangeSetBeingAvailableAsync(changeSetResponse.Id))
-                return false;
+            await WaitForChangeSetBeingAvailableAsync(changeSetResponse.Id);
```

Comment:
The `WaitForChangeSetBeingAvailableAsync` method was refactored to no longer return booleans and instead throw an exception if there was an error.